python/triton/runtime/jit.py

```diff
@@ -247,7 +247,13 @@ def _type_of(key):
         tys = {
             "bool": "i1",
             "float8e4nv": "fp8e4nv",
+            "float8_e4m3fn": "fp8e4nv",
```

Comment:
Is this a map from something like a torch type to triton type? Why are there multiple keys mapping to the same value?

Reply:
There are two ways to indicate types of input to triton kernels
1) provide a torch tensor, then its type is used
2) call `triton.reinterpret(torch_tensor, triton_type)`, in this way, the triton_type is used.

For types as the key, this is to cover all the types either in triton or in pytorch that might be used. For the right(value) in the table, these are the internal types used in triton. Since same data type may use different names, but in triton, we will map it to the same type. That is why multiple keys can map to the same value.

The added items here is to make fp8 torch tensor used in the same way as other types of torch tensor without calling `triton.reinterpret`.